Shared/YearShowCellNode.swift

```diff
@@ -178,12 +191,24 @@ public class YearShowCellNode : ASCellNode {
         )
         bottom.style.alignSelf = .stretch
         
+        var footer : ASStackLayoutSpec? = nil
```

Comment:
I'm not too happy with this, but maybe it's just due to the naming of the variables. I can change this to wrap up a new stack inside the `bottom` stack if you'd like.